packages/stripe-graphql/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@nhost/graphql-stripe",
+  "version": "0.0.1",
+  "description": "Stripe GraphQL API for Nhost",
+  "license": "MIT",
+  "keywords": [
+    "nhost",
+    "hasura",
+    "graphql",
+    "stripe"
+  ],
+  "author": "Nhost",
+  "homepage": "https://nhost.io",
+  "bugs": "https://github.com/nhost/nhost/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nhost/nhost.git"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "dev": "NODE_ENV=development ts-node-dev dev/server.ts",
+    "build": "tsc",
+    "test:watch": "vitest",
+    "test:coverage": "vitest run --coverage",
```

Comment:
I think we definitely need to start writing e2e asap, from a basic test application in the same way we did for `@nhost/nhost-js`, `@nhost/hasura-auth-js` and `@nhost/hasura-storage-js`